x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/errors/error_details.spec.ts

```diff
@@ -14,7 +14,7 @@ const start = '2021-10-10T00:00:00.000Z';
 const end = '2021-10-10T00:15:00.000Z';
 const errorDetailsPageHref = url.format({
   pathname:
-    '/app/apm/services/opbeans-java/errors/0000000000000000000000000Error%201',
+    '/app/apm/services/opbeans-java/errors/0000000000000000000000000Error%200',
```

Comment:
The test was navigating to the wrong error page where it didn't find any errors and broke. And it was trying to link on the discover link but didn't find it.